README.md

```diff
@@ -113,23 +185,45 @@ tox -e behave -D reuse_container=container_name
 
 ## Building
 
-The packaging for the UA client package (ubuntu-advantage-tools) is
-in-tree, so you can build the package the way you would normally build
-a Debian package:
+The packaging for the UA client packages ubuntu-advantage-tools and
+ubuntu-advantage-pro) is in-tree in the debian directory. So you can build the
```

Comment:
`)` is now unnecessary

Reply:
I rewrote my ramble here :/.
Thanks, it now says:
```
Creating ubuntu-advantage-tools and ubuntu-advantage-pro is created from the    
debian/control file in this repository. You can build the                       
package the way you would normally build a Debian package:       
```